Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -876,12 +878,22 @@ def _exec_editor_multitest(self, request: _pytest.fixtures.FixtureRequest,
         editor_utils.cycle_crash_report(run_id, workspace)
 
         results = {}
-        test_filenames_str = ";".join(editor_utils.get_testcase_module_filepath(test_spec.test_module) for test_spec in test_spec_list)
+        temp_batched_file = None
```

Comment:
The line `temp_batched_file = None` can be removed now because it's directly assigned in the next line.

Reply:
removed